at_secondary/at_secondary_server/lib/src/verb/handler/scan_verb_handler.dart

```diff
@@ -107,12 +111,19 @@ class ScanVerbHandler extends AbstractVerbHandler {
   }
 
   /// Returns the current atsign keys.
-  /// @param - atConnectionMetadata: Metadata of the inbound connection
-  /// @param - List<String>: List of keys from the secondary persistent store
-  /// @return - String: Returns the keys of current atsign
-  String _getLocalKeys(
-      InboundConnectionMetadata atConnectionMetadata, List<String?> keys) {
-    var keyString;
+  ///
+  /// **Parameters**
+  ///
+  /// [atConnectionMetadata] Metadata of the inbound connection.
+  ///
+  /// [keys] List of keys from the secondary persistent store.
+  ///
+  /// **Returns**
+  ///
+  /// Returns the list of keys of current atsign.
```

Comment:
Re documentation - what I was hoping for was a description here of what this method actually does - i.e. returns a filtered list of the input keys where the filtering depends on the type of authentication on the inbound connection

Reply:
@yahu1031 Is my comment above `returns a filtered list of the input keys where the filtering depends on the type of authentication on the inbound connection` a good summary of the purpose of this method? If so then please add to the method documentation

Reply:
Yes this is more appropriate